expenses/migrations/0001_initial.py

```diff
@@ -1,4 +1,4 @@
-# Generated by Django 4.0.3 on 2022-04-13 14:31
+# Generated by Django 4.0.3 on 2022-04-18 08:39
```

Comment:
This file is already in the main code. 
why did you create it again?

Reply:
I think I fixed it